api/leaderboard.js

```diff
@@ -51,12 +51,12 @@ async function discordUpdate (steamid, category) {
   const time = ticksToString(run.time);
 
   let emoji = "🎲";
-  let output = `**${user.name.replaceAll("\\", "\\\\").replaceAll("@", "\\@")}**`;
+  let output = `**${user.name.replaceAll(/[*@_~`#[\]()\-.>\\:]/g, "\\$&")}**`;
```

Comment:
I'd maybe just move the regex into a constant so that if it needs to change, it only needs to change in once place, but looks good otherwise

Reply:
The regex is used in two files, so I'm not sure where to declare the constant. Do we have a specific file for this purpose? As I see, `config.ts` declares a bunch of constants, but it doesn't feel the right place for this constant.